File: Task1/main.java
import java.util.Scanner;

public class main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = scanner.nextInt();
        scanner.close();
        for (int i = 1; i <= num; i++) {
            for (int j = 1; j <= i; j++)
                System.out.print((char)(num - j + (int)'A') + " ");
            System.out.print('\n');
        }
    }
}

File: Task1/main.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val num = scanner.nextInt()
    scanner.close()
    for (i in 1..num) {
        for (j in 1..i) {
            print((num - j + 'A'.code).toChar() + " ")
        }
        println()
    }
}